tgrun/pkg/runner/vmi/embed/testhelpers.sh

```diff
@@ -292,3 +292,114 @@ function pvc_uses_provisioner() {
     fi
     echo "pvc $pvc uses provisioner $provisioner"
 }
+
+# test_push_image_to_registry spawns a job to copy an image from docker.io into the local registry.
+# The job is created in the default namespace and it is deleted after the image is copied. This
+# function has a timeout of 5 minutes. Image is pushed to <internal_docker_addr>/test/test:latest.
+function test_push_image_to_registry() {
+    echo "Trying to copy an image to kURL internal registry"
+    local registry_addr
+    registry_addr=$(kubectl get svc -n kurl registry -o jsonpath="{.spec.clusterIP}" 2>/dev/null)
+    if [ -z "$registry_addr" ]; then
+        echo "Failed to get registry address"
+        exit 1
+    fi
+
+    kubectl create -f - <<EOF
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: skopeo-copy
+  namespace: default
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      volumes:
+      - name: authfile
+        secret:
+          secretName: registry-creds
+      containers:
+      - name: skopeo
+        image: quay.io/skopeo/stable:latest
+        volumeMounts:
```

Comment:
I would add an imagepullpolicy here (ifnotpresent) so that we can potentially use this in airgap tests

Reply:
(same with the other job)

Reply:
🤦 of course, this won't work in airgap because the source is the internet

Reply:
done.

Reply:
ops, yeah, that is true.